cleantalk.php

```diff
@@ -626,6 +626,8 @@ function apbct_activation__new_blog($blog_id, $_user_id, $_domain, $_path, $_sit
     require_once(CLEANTALK_PLUGIN_DIR . 'inc/cleantalk-public.php');
     require_once(CLEANTALK_PLUGIN_DIR . 'inc/cleantalk-public-integrations.php');
 
+    apbct_form__get_noCookie_data();
```

Comment:
Optimization: Thes function will calls update_option on every hit. Need to be reduced the update_option calling.